### PR TITLE
Adding Admin and User Routes

### PR DESCRIPTION
**Pull request by jrjohnson**
This is bare bones skeleton stuff to add the /admin /users and /users/:user_id routes.  I also added a global search component that currently only works for users, but should be able to work for many more things with time.

Need to fix up a few tests and get translation details.

### CODE REVIEW

File: app/locales/en/translations.js [hunk 1]
```diff
@@ -94,6 +94,7 @@ export default {
     'waitSaving': 'saving... one moment...',
     'select': 'Select',
     'invalidDatetimes': 'Invalid dates/times',
+    'globalSearchPlaceholder': 'Find Users...',
```
Comment (jrjohnson) [inline app/locales/en/translations.js]: @saschaben / @dartajax please translate.

File: app/locales/en/translations.js [hunk 2]
```diff
@@ -124,7 +125,8 @@ export default {
     'learnerGroups': 'Learner Groups',
     'courses': 'Courses and Sessions',
     'logo': 'Ilios Logo',
-    'menu': 'Ilios Menu'
+    'menu': 'Ilios Menu',
+    'admin': 'Admin',
```
Comment (jrjohnson) [inline app/locales/en/translations.js]: @saschaben / @dartajax please translate "Admin"

File: app/locales/en/translations.js [hunk 3]
```diff
@@ -320,6 +322,11 @@ export default {
     'badCredentials': 'Incorrect username or password',
     'noAccountExists': 'Your account {{accountName}} does not match any user records in Ilios. If you need further assistance, please contact your school’s Ilios administrator.',
   },
+  'user': {
+    'firstName': 'First Name',
+    'lastName': 'Last Name',
+    'manageUsers': 'Manage Users',
```
Comment (jrjohnson) [inline app/locales/en/translations.js]: @saschaben / @dartajax please translate "Manage Users"

File: app/locales/es/translations.js [hunk 4]
```diff
@@ -320,6 +320,10 @@ export default {
     'badCredentials': 'Incorrecta contraseña o nombre de usuario',
     'noAccountExists': 'Su cuenta no coincide con los registros de Ilios. Si usted necesita más asistencia, póngase en contacto con su administrador de Ilios',
   },
+  'user': {
+    'firstName': 'Nombre',
+    'lastName': 'Apellido',
```
Comment (jrjohnson) [inline app/locales/es/translations.js]: @dartajax please check google's work for first name and last name

File: app/locales/fr/translations.js [hunk 5]
```diff
@@ -320,6 +320,10 @@ export default {
     'badCredentials': "Nom d'utilisateur ou mot de passe incorrect",
     'noAccountExists': "Votre compte d'utilisateur {{accountName}} ne correspond pas à ceux des utilisateur enregistre en Ilios. Si vous avez besoin d'aide, n'hesitez pas à communiquer avec votre administrateur  d'Ilios.",
   },
+  'user': {
+    'firstName': 'Prénom',
+    'lastName': 'Nom de Famille',
+  },
```
Comment (jrjohnson) [inline app/locales/fr/translations.js]: @saschaben  please check google's work for first name and last name

File: app/locales/en/translations.js (hunk 1 above)
Comment (saschaben) [inline app/locales/en/translations.js]: Fr.:"Trouver des utilisateurs..."

File: app/locales/en/translations.js (hunk 2 above)
Comment (saschaben) [inline app/locales/en/translations.js]: Fr.: "Admin"

File: app/locales/en/translations.js (hunk 3 above)
Comment (saschaben) [inline app/locales/en/translations.js]: Fr.: "Gérer les utilisateurs"

File: app/locales/fr/translations.js (hunk 5 above)
Comment (saschaben) [inline app/locales/fr/translations.js]: Google is A-OK, but use sentence case not title case: "Nom de famille", and "Prénom".

**Comment by saschaben** [comment]
looking like a great start. is it possible to reverse the stacking of the live search control and the manage users button in their liquid container div? That would remove the "results push the button down the screen" issue with the live search control. I know it sort of mucks up the layout concept, but it's just a  thought. If you tried it already and it either doesn't fly or looks like crap, that's fine. Just want to cover that issue since we discussed it yesterday.

**Comment by jrjohnson** [comment]
It helps with the button pushing, but it would be temporary at best since more dashboard containers are going to end up on this page and they would get pushed down as well.  I was successful in limiting the space by adding scroll bars to the search results so it only takes up 500px of vertical space no matter how many results there are.

File: app/locales/en/translations.js (hunk 1 above)
Comment (dartajax) [inline app/locales/en/translations.js]: Es/Sp .. "Encontrar Usuarios..."

File: app/locales/en/translations.js (hunk 2 above)
Comment (dartajax) [inline app/locales/en/translations.js]: Spanish: "Administración"

File: app/locales/en/translations.js (hunk 3 above)
Comment (dartajax) [inline app/locales/en/translations.js]: Spanish: "Administrar Usuarios"

File: app/locales/es/translations.js (hunk 4 above)
Comment (dartajax) [inline app/locales/es/translations.js]: Names look fine.

**Pull request merged**